subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -16,34 +16,72 @@
+* Gradle uses various caches to reuse outputs from previous builds.
+* With a shared build cache, you can even reuse outputs from other machines.
 JVM foundation::
-Gradle runs on the JVM. This is a bonus for users familiar with Java, since build logic can use the standard Java APIs.
-It also makes it easy to run Gradle on different platforms.
+* Gradle runs on the JVM. This is a bonus for users familiar with Java, since build logic can use the standard Java APIs.
+* It also makes it easy to run Gradle on different platforms.
 Conventions::
-Gradle makes common types of projects easy to build through conventions. Plugins set sensible defaults to keep build scripts minimal.
-But these conventions don't limit you: you can configure settings, add your own tasks, and make many other customizations in your builds.
+* Gradle makes common types of projects easy to build through conventions. Plugins set sensible defaults to keep build scripts minimal.
+* But these conventions don't limit you: you can configure settings, add your own tasks, and make many other customizations in your builds.
 Extensibility::
-Most builds have special requirements that require custom build logic.
-You can readily extend Gradle to provide your own build logic with custom tasks and plugins.
-See Android builds for an example: they add many new build concepts such as flavors and build types.
+* Most builds have special requirements that require custom build logic.
+* You can readily extend Gradle to provide your own build logic with custom tasks and plugins.
+* Android builds add many new build concepts such as flavors and build types.
 IDE support::
-Several major IDEs provide interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, VSCode, and NetBeans.
-Gradle can also generate the solution files required to load a project into Visual Studio.
+* Several major IDEs provide interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, VSCode, and NetBeans.
+* Gradle can also generate the solution files required to load a project into Visual Studio.
 Insight::
-https://scans.gradle.com/[Build Scan™] provides extensive information about a build that you can use to identify issues.
-You can use Build Scans to identify problems with a build's performance and even share them for debugging help.
+* https://scans.gradle.com/[Build Scan™] provides extensive information about a build that you can use to identify issues.
+* You can use Build Scans™ to identify problems with a build's performance and even share them for debugging help.
+
+== User Manual
+
+The Gradle Build Tool user manual is broken down into the following sections:
+
+Quick Start :: Get up and running installing Gradle.
+Running Gradle Builds :: Use Gradle Build Tool to build your project.
+Authoring Gradle Builds :: Develop tasks and plugins to customize your build.
```

Comment:
For the time being I'm going to keep the existing menu / flow of the user manual. I agree that it needs to be updated but this will be done at a later time.